src/client/features/search/query-entity-info.js

```diff
@@ -98,15 +76,16 @@ const getNcbiInfo = ( ids, genes ) => {
     let geneResults = result.result;
 
     return geneResults.uids.map( gene => {
-      let originalSearch = ids[ gene ];
-      let links = idToLinkConverter( genes[ originalSearch ] );
+      const originalSearch = ids[ gene ];
+      const geneId = genes ? genes[ originalSearch ] : { 'NCBI Gene': originalSearch };
```

Comment:
Surely links to HGNC and Gene Cards can be generated from the `name` value in the response. They `genes` parameter only seems to possibly supply a uniprot accession.